driver-core/src/main/com/mongodb/connection/netty/NettyStreamFactoryFactory.java

```diff
@@ -99,6 +110,42 @@ public Builder eventLoopGroup(final EventLoopGroup eventLoopGroup) {
             return this;
         }
 
+        /**
+         * Sets the tuner for a {@linkplain SslContextBuilder#forClient() client-side} {@link SslContext io.netty.handler.ssl.SslContext},
+         * which overrides the standard {@link SslSettings#getContext()}.
+         * By default the tuner is {@code null} and {@link SslSettings#getContext()} is at play.
+         * <p>
+         * This option may be used as a convenient way to utilize
+         * <a href="https://www.openssl.org/">OpenSSL</a> as an alternative to the TLS/SSL protocol implementation in a JDK.
+         * To achieve this, specify {@link SslProvider#OPENSSL}
+         * {@linkplain SslContextBuilder#sslProvider(SslProvider) TLS/SSL protocol provider} via the tuner.
+         * Note that doing so adds a runtime dependency on
+         * <a href="https://netty.io/wiki/forked-tomcat-native.html">netty-tcnative</a>, which you must satisfy.
```

Comment:
There are multiple artifacts that may be used to satisfy this dependency. They differ by the target OS, ISA, native dependencies which are linked either dynamically or statically. As a result, I don't think we can specify this dependency in the [published POM](https://search.maven.org/artifact/org.mongodb/mongodb-driver-core/4.3.0-beta3/jar) as an optional runtime dependency.